SpatialGDK/Source/SpatialGDK/Public/EngineClasses/SpatialNetDriver.h

```diff
@@ -308,7 +327,13 @@ class SPATIALGDK_API USpatialNetDriver : public UIpNetDriver
 
 	void ProcessOwnershipChanges();
 
+	// Has a certain interval (in seconds) been passed since the previous timestamp
+	bool HasTimedOut(const float Interval, uint64& TimeStamp);
+
 	TSet<Worker_EntityId_Key> OwnershipChangedEntities;
 	uint64 StartupTimestamp;
 	FString StartupClientDebugString;
+
+	TMultiMap<Worker_EntityId, EActorMigrationResult> MigrationFailureLogStore;
```

Comment:
Failing the merge check as "TMap must have a hashable KeyType unless a custom key func is provided."
Will fix Monday morning.

Reply:
Use Worker_EntityId_Key instead of Worker_EntityId, an existing known issue with Linux builds as the latter is not supported by TMap.